deployment/helm/nri-plugins/balloons_values.yaml

```diff
@@ -0,0 +1,43 @@
+# Default values for nri-plugins.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+---
+image:
+  name: ghcr.io/containers/nri-plugins/nri-resource-policy-balloons
+  # tag, if defined will use the given image tag, otherwise Chart.appVersion will be used
+  tag: unstable
+  pullPolicy: IfNotPresent
+container:
+  name: nri-resource-policy-balloons
+
+namespace: kube-system
+
+configMap:
+  data:
+    ReservedResources:
+      cpu: 750m
+
+spec:
+  containers:
+    ports:
+    - containerPort: 8891
+      hostPort: 8891
```

Comment:
I wouldn't "steal" this port from the host. Isn't containerPort enough?

Reply:
I'm not sure, but again, this is based on the kustomize https://github.com/containers/nri-plugins/blob/main/deployment/base/nri-resource-policy/daemonset.yaml#L31 and the other deployment methods we already have. To be frank, I'm not sure if it is super necessary to expose the container port to the host. Perhaps it is needed for debugging (or instrumentation) but perhaps it is okay not to expose the port to the host by default and let the users decide if they wish.

WDYT @klihub @jukkar @marquiz @askervin

Reply:
@klihub I just checked removing the `hostPort` in my OpenShift based setup (because I still had it running), so not exposing host port didn't break the way TA plugin assigns resources to containers.